Cron/RetrieveReviews.php

```diff
@@ -2,90 +2,108 @@
 
 namespace Elgentos\Kiyoh\Cron;
 
+use Magento\Framework\Exception\NoSuchEntityException;
 use Magento\Framework\HTTP\Client\Curl;
 use Magento\Framework\Module\Declaration\Converter\Dom;
-use Magento\Framework\View\Element\Template\Context;
+use Magento\Variable\Model\Variable;
 use Magento\Variable\Model\VariableFactory;
 use \Psr\Log\LoggerInterface;
-use Magento\Store\Model\StoreManagerInterface;
+use Magento\Framework\Serialize\Serializer\Json;
+use Magento\Framework\Exception\CronException;
+use Elgentos\Kiyoh\Model\Config;
 
 class RetrieveReviews {
 
     /**
```

Comment:
If you set the typehint like `private Curl $curl`, you don't need the docblock anymore. So you can remove this everywhere.